### ISSUE TITLE
LibreOffice Write: nothing spoken when reviewing a document using up and down arrows

### ISSUE THREAD

**Issue by MichelSuch** [nvaccess/nvda#10536]
Using NVDA alpha-19266,67bf1df6, nothing is spoken when scrolling a document under libreOffice Writer 6.3.3. 
### Steps to reproduce:
1. Open a document using libreOffice writer.
2. Try to scroll it using the up and down arrows.
### Actual behavior:
The document content is not spoken, though it is correctly displayed if you have a braille display.
### Expected behavior:
The content of each line should be spoken when arrowing on it. Used to work with NVDA 2019.2.1.
#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
alpha-19266,67bf1df6
#### Windows version:
Windows 10Ver1909 (64-bit) build 18363.476
#### Name and version of other software in use when reproducing the issue:
LibreOffice 6.3.3 64 bits

**Comment by lukaszgo1** [comment]
This should be marked as a regression. cc @leonardder  @michaelDCurran

**Comment by leonardder** [comment]
@MichelSuch: Please fully follow the issue template, especially the last few questions are important here.

**Comment by lukaszgo1** [comment]
The most important question i.e. 
> Have you tried any other versions of NVDA? If so, please report their behaviors.

has been answered but not in the proper place in the template.
@MichelSuch said:
> Used to work with NVDA 2019.2.1.

**Comment by leonardder** [comment]
Thanks for pointing this out.

This definitely is a regression, caused by 4d1d4f640 (#10463).

I'm afraid the only thing we can do now is revert this and look at another solution for #4317. Alternative ideas @lukaszgo1?

**Comment by Brian1Gaff** [comment]
I posted some questions about this software in the devel group a few days 
ago, it seems not to work correctly in the old nvda either, ie  you get 
mangled test apparently in the file you are typing into and this also 
happened in the earlier release of Libra Write both installed and Portable. 
Sometimes also you cannot read the file list either.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- O

**Comment by Brian1Gaff** [comment]
Could this be at the bottom of the Utorrent issue recently posted here as 
well?
As I say, Liba Office has been plagued with weird things for some time now, 
sadly.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
-----

**Comment by lukaszgo1** [comment]
@leonardder wrote:

> Alternative ideas @lukaszgo1?



I have created #10537 to fix this. If you would be able to take a look that would be fantastic.
